pajbot/web/routes/base/login.py

```diff
@@ -166,3 +195,67 @@ def logout():
     @twitch.tokengetter
     def get_twitch_oauth_token():
         return session.get("twitch_token")
+
+    if not spotify:
+        return
```

Comment:
what does this do

Reply:
returns early so it doesnt register the pages.